napari/components/dims.py

```diff
@@ -145,33 +156,61 @@ def ndim(self, ndim):
             self._point = self._point[-ndim:]
             self._interval = self._interval[-ndim:]
             self._mode = self._mode[-ndim:]
-            self._display = self._display[-ndim:]
+            order = np.array(self._order[-ndim:])
+            order[np.argsort(order)] = list(range(len(order)))
+            self._order = list(order)
 
             # Notify listeners that the number of dimensions have changed
             self.events.ndim()
 
     @property
     def indices(self):
-        """Tuple of slice objects for slicing arrays on each dimension."""
+        """tuple of slice objects for slicing arrays on each dimension."""
```

Comment:
everything else is lowercase and tuple is lowercase in other places? i'm fine with uppercase but we should go through and systematically make everything the same rather than having a mix

Reply:
The first letter of the docstring should be uppercase. Everywhere in the Python ecosystem, docstrings are capitalised as in normal English prose.

Reply:
ok fixed in this file, but there will be other places we need to fix at another time. I've talked with @kne42 about her doing a pass through all our doc strings sometime soon